<launch>
    <group>
        <arg name="weights" default="yolov3.weights"/>
        <arg name="cfg" default="yolov3.cfg"/>  
        <arg name="labels" default="coco.txt"/>   

        <node pkg="spot_micro_perception" 
        type="spotMicroDetectionManager.py" 
        name="detection_publisher" 
        output="screen">
            <param name="model_weights" value="$(find spot_micro_perception)/configs/networks/$(arg weights)"/>
            <param name="model_cfg" value="$(find spot_micro_perception)/configs/networks/$(arg cfg)"/>
            <param name="model_labels" value="$(find spot_micro_perception)/configs/labels/$(arg labels)"/>
        </node>
    </group>
</launch>
